config/config.toml.example

```diff
@@ -191,7 +191,7 @@ cluster-ssl-key = ""
 # Configurations of the encryption method to use for encrypting the spilled data files.
 # Possible values are "plaintext", "aes128-ctr", if not set, it will be "plaintext" by default.
 # "plaintext" means encryption is disabled.
-spilled-file-encryption-method = "plaintext"
+data-encryption-method = "plaintext"
```

Comment:
If possible, we can reuse the config `cluster-ssl-ca`, if it is on, then the spilled data files are encrypted.

Reply:
cc @XuHuaiyu